Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8u212-jre-alpine
+FROM azul/zulu-openjdk-alpine:16.0.1-16.30.15-jre
```

Comment:
That does not seem as a mere replacement (to fix an issue with M1) but a version upgrade (that impacts all the other non-M1 systems). Is this intentional?

Reply:
It's intentional since version 8 was actually hanging and therefore an upgrade was needed. From my testing jdk11 had the same problem

Reply:
I have updated the image now to a java8 based image that works with the m1